scripts/submitHistoFiller.py

```diff
@@ -13,7 +13,8 @@
 FLAGS = parser.parse_args()
 
 #hourdate = datetime.datetime.now().strftime('%Y.%m.%d_%H.%M.%S').replace('.','-')
-outdir = "filljobs_" + FLAGS.tag
+outdir = "/afs/cern.ch/user/j/jowulff/Condor/res_HH/filljobs_" + FLAGS.tag
+condor_dir = "/afs/cern.ch/user/j/jowulff/Condor/res_HH/"
```

Comment:
this file is not used to run on lxplus right? if so could you restore it?

Reply:
Done!